clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -320,9 +320,10 @@ Changes in existing checks
 
 - Improved :doc:`readability-implicit-bool-conversion
   <clang-tidy/checks/readability/implicit-bool-conversion>` 
-   - `UseUpperCaseLiteralSuffix` : check by adding the option to select the case of the literal suffix in fixes and fixing false positive for implicit conversion of comparison result in C23.
-   - `CheckConversionsToBool`: check by adding the option to allow enabling or disabling warnings for implicit conversions to `bool` .
-   - `CheckConversionsFromBool`: check by adding the option to allow enabling or disabling warnings for implicit conversions from `bool`.
+   Check by adding the option 
```

Comment:
Belongs to previous line.

Reply:
Sorry but i havent got this one . Can you please clarify?

Reply:
Thanks got it now.